src/SprykerEco/Zed/Afterpay/Communication/Plugin/Checkout/Oms/Condition/IsAuthorizationCompletedPlugin.php

```diff
@@ -20,7 +20,6 @@
  */
 class IsAuthorizationCompletedPlugin extends AbstractPlugin implements ConditionInterface
 {
-
     const AUTHORIZE_TRANSACTION_ACCEPTED = AfterpayConstants::API_TRANSACTION_OUTCOME_ACCEPTED;
     /**
```

Comment:
`new line` is missing between constants and everything else (run `code:sniff` — should detect this)